content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,197 @@
+
+![Reference panel revamp](https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Reference%20Panel%20Image.png "Refernce panel revamp")
+
+[Docs](https://docs.sourcegraph.com/code_navigation/explanations/features#find-references)
+
+
+### 🏗️ High-leverage ways to improve your entire codebase
+
+<Badge text="Batch Changes" color="blue" size="small" onClick={() => window.open("/batch-changes", "_self")} />
+
+#### Make changes across all of your codebases at enterprise scale with server-side Batch Changes (beta)
+Batch Changes allows you to programmatically define changes across the codebase with a single declarative file. Now, changes can be run across thousands of repositories with a better development experience. Previously, you had to run src-cli locally to create batch changes, which could take an impractical amount of time or be brittle for large or resource-intensive batch changes. By running Batch Changes server-side you can:
+
+- Run large-scale or resource-intensive batch changes without clogging your local machine
+- Run large batch changes quickly by distributing them across an autoscaled pool of compute instances
+- Get a better debugging experience with the streaming of logs directly into Sourcegraph.
+
+This feature requires admins to set up executors (much like CI agents), which Sourcegraph will use to offload expensive tasks. Executors can also be used to run [Code Intelligence auto-indexing](https://github.com/sourcegraph/about/blob/bf341aafde75450c295aaec064470017020cdcd8/content/blogposts/2022/sourcegraph-4.0.md#TODO-link-to-section-or-docs) as well. One or more executors are available by default on Cloud instances of Sourcegraph.
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Batch_Changes_Side_by_Side',
+    mp4: 'blog/release-post/4.0/Batch_Changes_Side_by_Side'
+  }}
+  loop={true}
+  title="Server-side Batch Changes"
+  caption="Side by side comparison of deploying Batch Changes locally vs. server-side"
+/>
+
+[Docs](https://docs.sourcegraph.com/batch_changes/explanations/server_side)
+
+
+### ☁️ Dedicated Sourcegraph Cloud instances for enterprise
+
+<Badge text="Admin" color="violet" size="small" onClick={() => window.open("URL_GOES_HERE", "_self")} />
+
+#### Get started with Sourcegraph faster than ever
+After many years of building Sourcegraph into a secure and highly-scalable solution, we're ready to bring it to the cloud. Sourcegraph Cloud now offers dedicated, single-tenant instances of Sourcegraph, making it easier than ever for new teams to get started. You can read more in our announcement blog here[/blog/enterprise-cloud].
+
+[Docs](https://docs.sourcegraph.com/cloud)
+
+### 💖 Advanced admin capabilities
+
+<Badge text="Admin" color="violet" size="small" onClick={() => window.open("URL_GOES_HERE", "_self")} />
+
+#### Save time upgrading to Sourcegraph 4.0 with multi-version upgrades
```

Comment:
This phrasing is a bit confusing "help to reduce the burden and time required." Be direct about the improvement:

Sourcegraph 4.0 supports multi-version upgrades so you can upgrade to 4.0 directly from version 3.20 or higher. 

We can then explain why this is important + link to instructions for upgrading to 4.0.

Reply:
Asking @macraig  - will update once I hear back. Good points.

Reply:
Updated. Each section has documentation, and this section links to the docs for multi-version upgrades.